src/tribler/core/components/metadata_store/restapi/metadata_endpoint_base.py

```diff
@@ -55,16 +57,16 @@ def sanitize_parameters(cls, parameters):
             "first": int(parameters.get('first', 1)),
             "last": int(parameters.get('last', 50)),
             "sort_by": json2pony_columns.get(parameters.get('sort_by')),
-            "sort_desc": bool(int(parameters.get('sort_desc', 1)) > 0),
+            "sort_desc": parse_bool(parameters.get('sort_desc', 1)),
```

Comment:
With this new `parse_bool` function, let's use `True` and `False` as default values instead of `1` and `0`, as it looks clearer this way.